### ISSUE TITLE
CR process, use conventional comments? 

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#6474]
For consideration, what does the group think of conventional comments? https://conventionalcomments.org/

To some extent we do this with "nit" and "side-comment" prefixes. Given we are pure distributed team, writing down process and clarity are very important (“Working well remotely requires writing things down,”  https://business.linkedin.com/talent-solutions/blog/work-flexibility/2020/tips-for-managing-remote-workforce-during-coronavirus).

I'm wondering how seriously it would be worthwhile to experiment with this and if we would want to take a stab at a list of conventional comment labels.

**Comment by RoiEXLab** [comment]
I think we can definitely agree on the definition of some terms to indicate that a comment is non-blocking, but I don't think questions ("why did you do x when I would've done y?", when both approaches achieve the same goal in different ways) about code decisions need extra prefixes for example. I should be obvious that if the reviewer meant to submit a blocking comment they would've suggested "do y because x has those problems" instead.

Also I think marking compliments with "praise" is silly. If I like a piece of code because I think it's a good abstraction I write a nice comment, but it should be obvious that there's nothing to be done there.

TL;DR: Some prefix conventions could make sense for our use-case, but I don't think we need a 100% machine readable labeling process, just enough so the reviewer is understood correctly.

**Comment by DanVanAtta** [comment]
Thanks for feedback, was an interesting article. The process overhead and formalizing it I could agree is more overhead than we strictly need right now. There are times when the 'strength' of a recommendation/comment is hard to interpret (eg: is this just being mentioned? Or is this something the other person would really want done?).
